source/threads.tex

```diff
@@ -1669,7 +1669,7 @@
 Initializes \tcode{ssource}.
 The new thread of execution executes
 \begin{codeblock}
-invoke(auto(std::forward<F>(f)), get_stop_token(),
+invoke(auto(std::forward<F>(f)), get_stop_token(),  // see \ref{func.invoke}, \ref{thread.jthread.stop}
```

Comment:
We should not add a cross-reference for `get_stop_token`; it's right in this (or the next) section.

Reply:
Agreed.

It's a bit annoying that the comment is not a great place for the cross-reference, since it's not clear which bit of the code it refers to (forward? auto? invoke?). I wonder if an explicit note (e.g. "invoke is defined in \ref(...)") would be clearer -- thoughts

Reply:
trying "for invoke, see \ref{...}", does that look OK?